.github/workflows/build-desktop-apps.yml

```diff
@@ -75,6 +75,15 @@ jobs:
             latest*.yml
           retention-days: 3
 
+      - name: Upload suite-desktop production artifacts
+        if: matrix.os == 'macos-14'
+        uses: actions/upload-artifact@v4
+        with:
+          name: suite-desktop-mac-arm.dmg
```

Comment:
This will be the name of zip archive so i think its better to get rid of `.dmg` here.

Reply:
118aab4f06df9355ef31462c836d2ea6370f66aa